packages/@react-spectrum/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -0,0 +1,198 @@
+
+```tsx example
+function Example() {
+  return (
+    <Breadcrumbs>
+      <Item uniqueKey="Folder 1">Folder 1</Item>
+      <Item uniqueKey="Folder 2">Folder 2</Item>
+      <Item uniqueKey="Folder 3">Folder 3</Item>
+    </Breadcrumbs>
+  );
+}
+```
+
+In order to identify children when a user takes an action on a item,
+each child needs to have a `uniqueKey` prop. See [Events](#events) for more information.
+
+It is possible (but not suggested) to use alternative components as children,
+as long as those components have a `uniqueKey` prop and render children.
+
+```tsx example
+function Example() {
+  let ExampleItem = ({children}) => <span>{children}</span>;
+  return (
+    <Breadcrumbs>
+      <ExampleItem uniqueKey="Folder 1">Folder 1</ExampleItem>
+      <ExampleItem uniqueKey="Folder 2">Folder 2</ExampleItem>
+      <ExampleItem uniqueKey="Folder 3">Folder 3</ExampleItem>
+    </Breadcrumbs>
+  );
+}
+```
+
+### Internationalization
+
+In order to internationalize Breadcrumbs, the strings of all child items should be localized.
+
+## Events
+
+Use the `onAction` prop as a callback to handle press events on items.
+The `uniqueKey` prop from the selected item will be passed into the callback.
+
+```tsx example
+function Example() {
+  let folders = [
+    {id: 1, label: 'Folder 1'},
+    {id: 2, label: 'Folder 2'},
+    {id: 3, label: 'Folder 3'}
+  ];
+  let [folderId, setFolderId] = React.useState();
+  return (
+    <div>
+      <Breadcrumbs onAction={(a) => setFolderId(a)}>
+        {folders.map(f => <Item key={f.id} uniqueKey={f.id}>{f.label}</Item>)}
+      </Breadcrumbs>
+      <p>Last Selected Folder: {folderId}</p>
+    </div>
+  );
+}
+```
+
+## Props
+
+<PropTable component={docs.exports.Breadcrumbs} links={docs.links} />
+
+## Visual Options
+
+### Size
+
+The small size is useful to maintain Breadcrumbs while reducing the height required to give users context.
+
+```tsx example
+<Breadcrumbs size="S">
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+</Breadcrumbs>
+```
+
+The medium size is the default.
+```tsx example
+<Breadcrumbs size="M">
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+</Breadcrumbs>
+```
+
+The large size places emphasis on the selected item as a page title or heading.
+
+```tsx example
+<Breadcrumbs size="L">
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+</Breadcrumbs>
+```
+
+### Heading
+
+Use the `isHeading` prop to add a role of "heading" to the highlighted item.
+If a change to the aria-level is required, use the `headingAriaLevel` prop.
+
+```tsx example
+<Breadcrumbs isHeading headingAriaLevel={2}>
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+  <Item uniqueKey="Folder 3">Folder 3</Item>
+</Breadcrumbs>
+```
+
+### Visible Items
+
+Breadcrumbs truncate by default when there is not enough room to display all items.
```

Comment:
will truncate to max 4 items (thats default)... and with `maxVisibleItems="auto"` will truncate to fit container size (or window size) with no max default